vllm/config.py

```diff
@@ -273,20 +274,24 @@ class CacheConfig:
         gpu_memory_utilization: Fraction of GPU memory to use for the
             vLLM execution.
         swap_space: Size of the CPU swap space per GPU (in GiB).
+        cache_dtype: Data type for kv cache storage.
     """
 
     def __init__(
         self,
         block_size: int,
         gpu_memory_utilization: float,
         swap_space: int,
+        cache_dtype_str: str,
```

Comment:
Let's just call it `cache_dtype`? The `_str` suffix seems unnecessary to me.

Reply:
Sure. Fixed.